src/i18n/kr.js

```diff
@@ -16,10 +19,11 @@ export default {
       'lock.unauthorized': '권한이 없습니다. 다시 시도해 주세요.',
       'lock.mfa_registration_required': '다단계 인증이 필요하지만, 디바이스가 등록되어 있지 않습니다. 먼저 디바이스를 등록하여 주세요.',
       'lock.mfa_invalid_code': '유효하지 않은 코드입니다. 다시 시도해 주세요.',
-      password_change_required: '처음 로그인하거나, 또는 비밀번호의 기한이 만료되어 비밀번호를 갱신해야 합니다.', // TODO: verify error code
+      password_change_required: '처음 로그인하거나, 또는 비밀번호의 기한이 만료되어 비밀번호를 갱신해야 합니다.',
       password_leaked: '본 계정은 외부 접근으로 인한 회원 정보 유출이 우려되어, 보안 상의 문제로 비활성화되었습니다. 계정을 다시 활성화하는 방법은 자동 전송된 이메일에 안내되어 있습니다.',
       too_many_attempts: '본 계정은 단시간에 복수의 로그인 시도가 감지되어 차단되었습니다.',
-      session_missing: '인증 요청이 완료되지 않았습니다. 열려있는 모든 다이얼로그 창을 닫고 다시 시도해 주시기 바랍니다.'
+      session_missing: '인증 요청이 완료되지 않았습니다. 열려있는 모든 다이얼로그 창을 닫고 다시 시도해 주시기 바랍니다.',
+      'hrd.not_matching_email': 'Please, use your corporate email to login.'
```

Comment:
This one is not korean 👎. The key seems to have quotes `'` and that's somehow breaking the script?